handbook/data-stores/mongodb.md

```diff
@@ -0,0 +1,133 @@
+
+Before you configure a remote MongoDB instance to use with SystemLink, ensure you have the following:
+
+- A server running SystemLink 2021 R1 or greater. Refer to [Installing and Configuring SystemLink Server and Clients](https://www.ni.com/documentation/en/systemlink/latest/setup/configuring-systemlink-server-clients/) for the basics of setting up a SystemLink server.
+- A standalone server running MongoDB, multiple servers hosting a MongoDB replica set, or a MongoDB Atlas Account
+- A user with the `readWriteAnyDatabase` or similar role. Refer to [Role-Based Access Control](https://docs.mongodb.com/manual/core/authorization/) for details. You reference this user in the MongoDB connection fields or connection string in **NI SystemLink Server Configuration**.
+
+!!!important
+    Due to a bug in a third-party MongoDB driver, SystemLink cannot connect to MongoDB instances where the MongoDB username either the `-` or the `_` character.
+
+## Single Node vs Multi Node MongoDB Deployments
+
+The following table summarizes when to use the various supported configurations of MongoDB with SystemLink.
+
+| Diagram | SystemLink + MongoDB Configuration | When to use |
+| | ----------- | ----------- |
+|<img src="../../img/single-box.png" width="1500px"/>| Single node with defaults     | You have one server hosting both MongoDB and SystemLink. Use this during evaluation, when working with less than 10 managed nodes, and in deployments that don't need the Asset Manager and Test Insights modules.
+|<img src="../../img/single-box.png" width="1500px"/>| Single node with increased index cache   | You have one server hosting both MongoDB and SystemLink. Use this during evaluation, in moderate deployments of less than 25 managed nodes, and for moderate usage of the Test Insights module producing less than 100 test steps and results each day.  |
+|<img src="../../img/mdb-sa.png" width="1500px" />| Multi node with a single standalone MongoDB Instance | You have SystemLink and MongoDB hosted on two separate servers. Use this configuration when the CPU, memory, and disk consumption of the MongoDB Windows service is impacting the operation of SystemLink services. While this configuration provides for greater reliability by splitting the servers running SystemLink and MongoDB, it does not provide greater redundancy for data storage.   |
+|<img src="../../img/mdb-rs.png" width="1500px"/>| Multi node with MongoDB replica sets | You have one dedicated server hosting SystemLink and three or more servers hosting MongoDB. **NI recommends this configuration for all production deployments including small and moderately sized deployments.** Use this to enable greater scale (>50 managed nodes, >1000 test steps and results daily). This configuration mitigates data loss with redundant replica sets. Refer to [MongoDB documentation](https://docs.mongodb.com/manual/tutorial/deploy-replica-set/) for setting up and hosting a MongoDB instance with replica sets. |
```

Comment:
Same comment as before on mixing steps and results. Steps are an order of magnitude larger than results.